Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -358,22 +358,23 @@ public async Task<ActionResult> UploadPicture(IFormFile file, string userId)
         /// <br/>200 OK on success
         /// </returns>
         [Authorize]
-        [HttpGet("newsfeed")]
+        [HttpPost("newsfeed")]
```

Comment:
why would this be a POST? just because you have a list you want to receive? that's not right at all. you can use [query string](https://docs.microsoft.com/en-us/aspnet/core/mvc/models/model-binding?view=aspnetcore-5.0#collections) for that kind of thing. but we don't want the client to have to send that anyway.

Reply:
Followers was implemented in the profile section. We are all trying to figure out a way we can work around this. Any suggestions on how we can fix this issue? We don't think it will be a good idea to use a get request and pass it in a query string.

Reply:
why wouldn't that be a good idea?

Reply:
there are tradeoffs here so i suppose we can use POST, but you should be able to explain why. [for example](https://stackoverflow.com/a/31984477/624710)